src/v2/guide/events.md

```diff
@@ -305,7 +305,7 @@ Vue.config.keyCodes.f1 = 112
 `.exact` 修飾子はイベントを引き起こすために必要なシステム修飾子の正確な組み合わせを制御します。
 
 ``` html
-<!-- これは Alt キー または Shift キーを押された場合でも、発行されます -->
+<!-- これは Ctrl キー または Shift キーを押された場合でも、発行されます -->
```

Comment:
原文は`this will fire even if Alt or Shift is also pressed` です。
この`also`の意味がうまく訳されてないので、現状わかりにくい文章になっちゃってますね。。
ここでいいたいのは、`.exact`修飾子を入れないと、Ctrl単体ではなく、Ctrl+AltでもCtrl+Shiftでもeventが起きますよ〜ということです。

これをふまえると、
`これは Alt キー または Shift キーを同時に押された場合でも、発行されます`
なんかが良いのではないかと思います！